src/less/section-notice/section-notice.stories.js

```diff
@@ -19,84 +19,93 @@ export const confirmation = () => `
         </svg>
     </div>
     <div class="section-notice__main">
-        <p>Congrats! You are the highest bidder!</p>
+        <p>Your price has been updated.</p>
     </div>
     <div class="section-notice__footer">
-        <a href="https://www.ebay.com" class="fake-btn fake-btn--secondary fake-btn--transparent">Action</a>
+        <button class="fake-link fake-link--action">Dismiss</button>
     </div>
+
 </div>
 `;
 
 export const RTL = () => `
 <div dir="rtl">
     <div class="section-notice section-notice--confirmation" role="region">
+
         <div class="section-notice__header" role="region" aria-roledescription="Notice">
             <svg aria-hidden="true" focusable="false" class="icon--confirmation-filled">
                 <use xlink:href="#icon-confirmation-filled"></use>
             </svg>
         </div>
         <div class="section-notice__main">
-            <p>Congrats  You are the highest bidder</p>
+            <p>Your price has been updated</p>
         </div>
         <div class="section-notice__footer">
-            <a href="https://www.ebay.com" class="fake-btn fake-btn--secondary fake-btn--transparent">Action</a>
+            <button class="fake-link fake-link--action">Dismiss</button>
         </div>
     </div>
 </div>
 `;
 
 export const attention = () => `
 <div class="section-notice section-notice--attention" role="region">
+
     <div class="section-notice__header" role="region" aria-roledescription="Notice">
         <svg aria-hidden="true" focusable="false" class="icon--attention-filled">
             <use xlink:href="#icon-attention-filled"></use>
         </svg>
     </div>
     <div class="section-notice__main">
-        <p>Something went wrong. Please try again.</p>
+        <p>Add required aspects to keep item live.</p>
     </div>
+
     <div class="section-notice__footer">
-        <a href="https://www.ebay.com" class="fake-btn fake-btn--secondary fake-btn--transparent">Action</a>
+        <button class="fake-link fake-link--action">Add</button>
     </div>
 </div>
 `;
 
 export const information = () => `
 <div class="section-notice section-notice--information" role="region">
+
     <div class="section-notice__header" role="region" aria-roledescription="Notice">
         <svg aria-hidden="true" focusable="false" class="icon--information-filled">
             <use xlink:href="#icon-information-filled"></use>
         </svg>
     </div>
     <div class="section-notice__main">
-        <p>Try refining your search term for more results.</p>
+        <p>Send offers to interested buyers.</p>
     </div>
+
     <div class="section-notice__footer">
-        <a href="https://www.ebay.com" class="fake-btn fake-btn--secondary fake-btn--transparent">Action</a>
```

Comment:
Page notice must also support anchor tag for its call to action, not just a button.